chromium/background-scripts/background.js

```diff
@@ -636,7 +642,7 @@ function onHeadersReceived(details) {
 
 // Registers the handler for requests
 // See: https://github.com/EFForg/https-everywhere/issues/10039
-chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, {urls: ["*://*/*"]}, ["blocking"]);
+chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, {urls: ["<all_urls>"]}, ["blocking"]);
```

Comment:
Is there a reason to use `<all_urls>` rather than enumerate `["*://*/*", "ftp://*/*"]`?

Reply:
Firefox doesn't seem to care either way, and never triggers this hook even with FTP urls.  I think you're right though, and enumerating both is a more constrained criteria than just `<all_urls>`.